src/Footer/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+
+
+function Footer(){
+    return<div className='footerMobile' >
+        <img  className='footerImg' src='https://4.bp.blogspot.com/-G_RMOhOyYuo/VstyE_1eAuI/AAAAAAAAC_0/FSxpj2usLmQ/s1600/mars%2B55.png'/>
```

Comment:
As we want this image on all pages, it would be nicer if we just took this photo and put it in our app so that we don't need to download the image from the internet every time we want to display a page.  you might need to google adding images to react-app but you'll need to put it inside the /public folder somewhere :) maybe public/images

Reply:
We have added the footer mars horizon image to the public folder and added a background colour to the footer :)